psi4/src/psi4/libfock/hamiltonian.cc

```diff
@@ -248,20 +248,20 @@ void CISRHamiltonian::product(const std::vector<std::shared_ptr<Vector> >& x,
                 int nvir = Cavir_->colspi()[h ^ symm];
                 for (int i = 0; i < nocc; i++) {
                     for (int a = 0; a < nvir; a++) {
-                        double x = xp[i * nvir + a + offset];
-                        if (x != 0.0 && x != 1.0) {
+                        double local_x = xp[i * nvir + a + offset];
+                        if (local_x != 0.0 && local_x != 1.0) {
```

Comment:
So this is where name `x` (a `const` arg to the fn) was being reused. Anything clever that I'm missing by suggesting `local_x` as more readable. And appeasing lgtm